src/tests/accounts/index.spec.ts

```diff
@@ -1,19 +1,115 @@
 import { expect } from 'chai';
 import server from '../../index';
 import request from 'supertest';
-import { dummyAccount } from '../generators/account-generator';
+import { dummyAccount, getToken } from '../generators/account-generator';
 
-describe(`Account test suite.`, () => {
-  it(`Successful sign up.`, async () => {
-    const data = dummyAccount();
-    const signUpMutation = `
+const signUpMutation = `
     mutation($input:SignUpInput!) {
         signUp(input: $input){
             token
         }
     }`;
 
+const authenticatepMutation = `
+      mutation($input:AuthenticateInput!){
+        authenticate(input: $input){
+              token
+          }
+      }
+  `;
+
+const meQuery = `
+  query{
+      me{
+          lastName
+          firstName
+          emailAddress
+        }
+  }
+`;
+
+let token: string = ``;
```

Comment:
This makes me believe that your test scenarios are sharing the same account. As much as possible, your tests should be completely isolated and does not have any dependencies from one another.